### DISCUSSION TITLE
Trial account downgraded to Hobby

### DISCUSSION THREAD

**Discussion by Jodi717** [vercel/community#3343] [Help]
### Summary

My developer and I applied our activation key. He is getting ready to deploy our build and we're seeing that its inactive and I"m not able to access the account, only my hobby account. He did build his code on his own account and intended to move it to mine when it was ready for release. 

I can't even get support on this because both of our accounts are "hobby accounts"
This is really important.
Jodi

### Steps to Reproduce

Click on Vercel.com, 
Account is not available. 

Developer goes my build in his account and it has the activation key but indicates that its inactive.

**Comment by amyegan** [comment]
Hey @Jodi717. I'm not sure I fully understand the situation. Can you share screenshots of what your developer sees? I want to be sure I'm looking for the right thing.

If you let me know the team id or team slug, then I can also look for more information about the account to help us figure out what is going wrong.

**Comment by Jodi717** [comment]
See attached - code at the bottom of the screen.I didn’t want to post this Sent from my iPhone please excuse typos and unexpected auto corrections On Jul 24, 2023, at 1:10 PM, Amy ***@***.***> wrote:﻿
Hey @Jodi717. I'm not sure I fully understand the situation. Can you share screenshots of what your developer sees? I want to be sure I'm looking for the right thing.
If you let me know the team id or team slug, then I can also look for more information about the account to help us figure out what is going wrong.

**Reply by amyegan** [reply]
It looks like you replied by email, but the images/code did not make it through to GitHub. Would you be able to share it in a comment directly in this discussion?

**Comment by Jodi717** [comment]
https://github.com/orgs/vercel/discussions/3343#discussioncomment-6533851
![image (30)](https://github.com/vercel/community/assets/119544848/e7dc0ff7-bb4d-4336-803f-755fce291534)

**Comment by Hasham-dev** [comment]
Hello, @amyegan ! Let me provide some additional context by following up on @Jodi717's issue.

I'm currently working on the frontend of our team, and recently @Jodi informed me that she received an email from your representative. The email suggested creating our team on Vercel and by sharing the team ID, the representative would activate a 6-month subscription (@Jodi717  please attach those mail screenshot here).

As instructed, we created the team and provided the ID to the representative. They responded, confirming that the account was activated. However, I've noticed a banner indicating that our Pro Trial has ended and we need to upgrade in order to continue using the features, restricting the team to only one person.

It seems the representative we've been in contact with may be on leave until the 25th, which is why we hoped the issue could be resolved here.

**Reply by amyegan** [reply]
Thanks for this additional clarification. I believe the team owner needs to add a payment method to complete the upgrade from the trial to a regular team so the credit can apply as a discount to the monthly bill. But I'm checking with the growth team now to make sure your credit is applied correctly so you don't have any problems.

**Comment by Jodi717** [comment]
Hi @amyegan 
Attached is a screen shot of the email confirmation


![vercel confirm](https://github.com/vercel/community/assets/119544848/3bda64ad-9f70-4be6-95d5-8317ae9673c6)

**Comment by amyegan** [comment]
@Jodi717 @Hasham-dev Thank you both for reaching out and walking me through your situation. I confirmed with the growth team that you'll need to add a payment method and upgrade the team to Pro so the credit can be applied. Then the credit will apply up to $200 per month for 6 months, and the card will only be charged when the credit expires or in case of any usage overage or add-on features that make the monthly bill higher than the monthly credit limit.

Sorry for the confusion and frustration you have had with this process. Your original contact is back from his time off and he will be in contact soon to make sure your Pro account activation goes smoothly this time around.

**Comment by Hasham-dev** [comment]
@amyegan Thanks for clearing thing out, but @Jodi717 need to add payment method but right now the team we created is only showing to me.

Can you do something at your end to transfer that team ID to @Jodi717 vercel account ? so then she can add payment method and add more team members if required

**Reply by amyegan** [reply]
I checked on your account today and it looks like Gurjot already got Jodi access to the team and re-added your credits. Please let us know if there is anything else you need 🙂

**Comment by Jodi717** [comment]
Thank you so much for your help. We’re back on trackSent from my iPhone please excuse typos and unexpected auto corrections On Jul 26, 2023, at 5:28 PM, Amy ***@***.***> wrote:﻿
I checked on your account today and it looks like Gurjot already got Jodi access to the team and re-added your credits. Please let us know if there is anything else you need 🙂
